services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: MovieSession) -> QuerySet:
```

Comment:
The parameter `movie_session_id` is expected to be an integer, but the type hint `MovieSession` suggests it should be a `MovieSession` object. It should be `movie_session_id: int`.